main.go

```diff
@@ -134,8 +137,13 @@ func resizeImage(src, dst string, width, height uint) {
         log.Panicln(err)
     }
 
+    err = mw.SetImageFormat(outputFormat)
+    if err != nil {
+        log.Panicln(err)
+    }
+
     // Set the compression quality to 95 (high quality = low compression)
-    err = mw.SetImageCompressionQuality(95)
+    err = mw.SetImageCompressionQuality(70)
```

Comment:
Size reduction is noticeable and we have a black to transparent gradient on top of the image in the landing pages header so quality is not as crucial. But yes, I think the right way to do it is to pass it as another argument in the url.